fili-core/src/test/groovy/com/yahoo/bard/systemconfig/ModuleSetupSpec.groovy

```diff
@@ -0,0 +1,31 @@
+// Copyright 2016 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.systemconfig
+
+import com.yahoo.bard.webservice.config.ConfigResourceLoader
+import com.yahoo.bard.webservice.config.SystemConfig
+
+import org.apache.commons.configuration.Configuration
+
+import spock.lang.Specification
+
+abstract class ModuleSetupSpec extends Specification {
+
+    public static final String MODULE_NAME_KEY = "moduleName"
+    public static final String MODULE_RESOURCE_NAME = "/moduleConfig.properties"
+
+    def "Test that SystemConfig loads for this module"() {
```

Comment:
1. "Test that" isn't helpful in the test name.
2. It would be cool if the module name could be injected into here somehow? Perhaps via a `where` block that gets it's value from the `getModuleName` method?